recipes/r-pathfindr.data/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set version = '2.0.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pathfindr.data
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/pathfindR.data_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pathfindR.data/pathfindR.data_{{ version }}.tar.gz
+  sha256: eb76a8c611010b6cae6003cda4eba9f2ec249e7c6113bf67039a9a46944df89d
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  noarch: generic
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('pathfindR.data')"           # [not win]
+    - "\"%R%\" -e \"library('pathfindR.data')\""  # [win]
+
+about:
+  home: https://github.com/egeulgen/pathfindR.data
+  license: MIT
+  summary: This is a data-only package, containing data needed to run the CRAN package 'pathfindR',
+    a package for enrichment analysis utilizing active subnetworks. This package contains
+    protein-protein interaction network data, data related to gene sets and example
+    input/output data.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+  doc_url: https://github.com/egeulgen/pathfindR.data
+  dev_url: https://github.com/egeulgen/pathfindR.data
+extra:
+  recipe-maintainers:
+    - egeulgen
+
```

R packages should include Conda Forge's R team:

```suggestion
    - conda-forge/r

```